packages/dx-grid-core/src/plugins/table-column-resizing/computeds.ts

```diff
@@ -1,21 +1,44 @@
 import { TABLE_DATA_TYPE } from '../table/constants';
 import { TableColumn, SpecifyWidthsFn, TableColumnsWithWidthFn } from '../../types';
+import { isValidValue } from './helpers';
+
+const VALID_UNITS = ['px', '%', 'em', 'rem', 'vm', 'vh', 'vmin', 'vmax', ''];
```

Comment:
This constant is same with `VALID_UNITS` from a `helpers` file.

Reply:
We don't have `VALID_UNITS` in `table-column-resizing/helpers`. We define some `VALID_UNITS` in `helpers.test` for some tests.